<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptWidget</class>
 <widget class="QMainWindow" name="EncryptWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncryptWidget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/openssl.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_dec_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Расшифровать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enc">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>220</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Зашифровать файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите файл:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectFile">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите пароль</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
